dlp/snippets/deid/deid_test.go

```diff
@@ -263,3 +271,127 @@ func TestDeIdentifyWithWordList(t *testing.T) {
 		t.Errorf("deidentifyWithWordList(%q) = %q, want %q", input, got, want)
 	}
 }
+
+func TestDeIdentifyDeterministic(t *testing.T) {
+	tc := testutil.SystemTest(t)
+
+	input := "Jack's phone number is 2255338899"
+	infoTypeNames := []string{"PHONE_NUMBER"}
+	keyRingName, err := createKeyRing(t, tc.ProjectID)
+	if err != nil {
+		t.Fatal(err)
+	}
+	keyFileName, cryptoKeyName, keyVersion, err := createKey(t, tc.ProjectID, keyRingName)
+	if err != nil {
+		t.Fatal(err)
+	}
+	surrogateInfoType := "PHONE_TOKEN"
+	want := "output : Jack's phone number is PHONE_TOKEN(36):"
+
+	var buf bytes.Buffer
+
+	if err := deIdentifyDeterministicEncryption(&buf, tc.ProjectID, input, infoTypeNames, keyFileName, cryptoKeyName, surrogateInfoType); err != nil {
+		t.Errorf("deIdentifyDeterministicEncryption(%q) = error '%q', want %q", err, input, want)
+	}
+
+	if got := buf.String(); !strings.Contains(got, want) {
+		t.Errorf("deIdentifyDeterministicEncryption(%q) = %q, want %q", input, got, want)
+	}
+
+	defer destroyKey(t, tc.ProjectID, keyVersion)
```

Comment:
issue(blocking): move this call to `defer` right to line 288. It's best practice to keep a destructor like this closer to what it's destroying.

Reply:
got your point, fixed it.